changelogs/fragments/6117-remove-stormssh-depend.yml

```diff
@@ -0,0 +1,2 @@
+trivial:
+  - ssh_config - Added StormSSH's config parser into ssh_config directly (https://github.com/ansible-collections/community.general/pull/6117).
```

Comment:
```suggestion
  - ssh_config - vendored StormSSH's config parser to avoid having to install StormSSH to use the module (https://github.com/ansible-collections/community.general/pull/6117).
```